bot/resources/tags/string-formatting.md

```diff
@@ -0,0 +1,27 @@
+**String Formatting Mini Language**
+The String Formatting Language in Python is a powerful way to tailor the display of strings and other data structures. This string formatting mini language works for f-strings and .format().
```

Comment:
Can you mention that it seems to have come from the usefulness of old-style `%s` formatting?

That's at least my take-away from [PyFormat](pyformat.info)

Reply:
What useful information does that provide to people? I'm not sure how knowing it derives from `%s` is necessarily useful for this tag.

Reply:
There are still uses for `%s`, like with logging when you need lazily evaluated strings, and knowing that you have similar options with `%s` could prove helpful?

That said I am pretty neutral on the matter, but thought I'd bring it up for judgement haha